tests/jerry/regression-test-issue-4189.js

```diff
@@ -0,0 +1,20 @@
+// Copyright JS Foundation and other contributors, http://js.foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+try {
+    eval("1 ; export default await 1")
```

Comment:
This is not a module test.

Reply:
Can you please explain what you mean? This is supposed to be a simple regression test, what I want to test is that the above code throws `SyntaxError`.

Reply:
This should be a syntax error, but I think `export default` is a syntax error, so this test does not test the `await`.

Reply:
I checked, you're right, this test is indeed incorrect.